readme.md

```diff
@@ -1,151 +1,131 @@
 ![Mend Renovate CLI banner](https://docs.renovatebot.com/assets/images/mend-renovate-cli-banner.jpg)
 
-# Renovate
-
-Automated dependency updates.
-Multi-platform and multi-language.
-
 [![License: AGPL-3.0-only](https://img.shields.io/badge/license-%20%09AGPL--3.0--only-blue.svg)](https://raw.githubusercontent.com/renovatebot/renovate/main/license)
 [![codecov](https://codecov.io/gh/renovatebot/renovate/branch/main/graph/badge.svg)](https://codecov.io/gh/renovatebot/renovate)
 [![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
 [![Build status](https://github.com/renovatebot/renovate/actions/workflows/build.yml/badge.svg)](https://github.com/renovatebot/renovate/actions/workflows/build.yml)
 ![Docker Pulls](https://img.shields.io/docker/pulls/renovate/renovate?color=turquoise)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/renovatebot/renovate/badge)](https://securityscorecards.dev/viewer/?uri=github.com/renovatebot/renovate)
 
-## Why Use Renovate?
+# What is the Mend Renovate CLI?
+
+Renovate is an automated dependency update tool.
+It helps to update dependencies in your code without needing to do it manually.
+When Renovate runs on your repo, it looks for references to dependencies (both public and private) and, if there are newer versions available, Renovate can create pull requests to update your versions automatically.
+
+## Features
+
+- Delivers update PRs directly to your repo
+  - Relevant package files are discovered automatically
+  - Pull Requests automatically generated in your repo
+- Provides useful information to help you decide which updates to accept (age, adoption, pass rates, merge confidence)
```

Comment:
```suggestion
- Renovate's PRs have badges to help you decide if you want to update:
  - The age of the package update
  - How many of the Mend-hosted app users updated to this version of the package
  - The test pass rates for Mend-hosted app users
  - A merge confidence score, that combines all the info above into a single number
```

Reply:
For the page re-write, I went with an approach to reduce the overall page height by deliberately collapsing bulleted lists into comma-separated strings.